storagenode/trust/service.go

```diff
@@ -36,10 +35,11 @@ type Pool struct {
 type satelliteInfoCache struct {
 	mu       sync.Mutex
 	identity *identity.PeerIdentity
+	nodeURL  storj.NodeURL
 }
 
 // NewPool creates a new trust pool using kademlia to find certificates and with the specified list of trusted satellites.
-func NewPool(kademlia *kademlia.Kademlia, trustAll bool, trustedSatelliteIDs string) (*Pool, error) {
+func NewPool(kademlia *kademlia.Kademlia, trustAll bool, trustedSatelliteURLs storj.NodeURLs) (*Pool, error) {
```

Comment:
`trustedSatellites`, since we don't have a string anymore we can rely on type annotations for names